tools/ete/test-data/11.nhx

```diff
@@ -0,0 +1 @@
+(((brat1_rattusnorvegicus,brat1_musmusculus),(brat1_homosapiens,brat1_pantroglodytes)),(brat1_susscrofa,brat1_canisfamiliaris));
```

Comment:
It seems all the output files have the wrong name, this should be `11_genetree.nhx`, no?